lib/components/project/user/list.vue

```diff
@@ -0,0 +1,226 @@
+<!--
+Copyright 2017 ODK Central Developers
+See the NOTICE file at the top-level directory of this distribution and at
+https://github.com/opendatakit/central-frontend/blob/master/NOTICE.
+
+This file is part of ODK Central. It is subject to the license terms in
+the LICENSE file found in the top-level directory of this distribution and at
+https://www.apache.org/licenses/LICENSE-2.0. No part of ODK Central,
+including this file, may be copied, modified, propagated, or distributed
+except according to the terms contained in the LICENSE file.
+-->
+
+<!-- Although Backend supports more complex use cases, we assume in this
+component that each that each user is assigned only one role and that further,
+each user either is a Project Manager or has no role. -->
+<template>
+  <div id="project-user-list">
+    <p id="project-user-list-heading">
+      The assigned Project Managers for this Project will be able to perform any
+      administrative or auditing task related to this Project. Sitewide
+      Administrators are automatically considered Managers of every Project. To
+      learn more about Projects and Managers, please see
+      <doc-link to="central-projects/#project-managers">this article</doc-link>.
+    </p>
+    <form id="project-user-list-search-form" @submit.prevent>
+      <!-- When search is disabled, we hide rather than disable this button,
+      because Bootstrap does not have CSS for .close[disabled]. -->
+      <button v-show="q != '' && !searchDisabled" type="button" class="close"
```

Comment:
Should we make this a darker color? If so, what should the hover/etc. colors be?